programs/rodeos/cloner_plugin.cpp

```diff
@@ -42,6 +42,7 @@ struct cloner_config : ship_client::connection_config {
    eosio::name filter_name = {}; // todo: remove
    std::string filter_wasm = {}; // todo: remove
    bool        profile = false;
+   uint32_t    force_write_stride = 200;
```

Comment:
Since this is always set this should be initialized to 0. Should have only one place where the default is set.

Reply:
Changed to zero.